src/workflow/workers/transcription.py

```
@@ -143,20 +174,25 @@ def _transcribe_episode(self, episode: Episode) -> Optional[str]:
 
         logger.info(f"Transcribing episode: {episode.title}")
 
-        # Get the Whisper model and transcribe
+        # Get the faster-whisper model and transcribe
         model = self._get_model()
-        result = model.transcribe(
-            audio=episode.local_file_path,
+        segments, info = model.transcribe(
```

Ruff (0.14.7)

Unpacked variable `info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)